pkg/pillar/diskmetrics/usage.go

```diff
@@ -1,12 +1,14 @@
-// Copyright (c) 2018-2019 Zededa, Inc.
+// Copyright (c) 2018-2023 Zededa, Inc.
```

Comment:
I guess you wanted to raise this to 2024?

Reply:
PR is getting old and mature ;-) Fixed.